src/wirecloud/platform/iwidget/utils.py

```diff
@@ -104,10 +105,10 @@ def update_screen_size_value(model: WidgetPositionsConfig, data: LayoutConfig, f
     value = getattr(data, field)
 
     if type(value) is not int:
-        raise TypeError(f'Field {field} must contain a number value')  # TODO translate this
+        raise TypeError(_(f'Field {field} must contain a number value'))
 
     if value < -1:
-        raise ValueError(f'Invalid value for {field} field')  # TODO translate this
+        raise ValueError(_(f'Invalid value for {field} field'))
```

Comment:
No puede haber traducciones con reemplazo antes de traducir, debe ser:
```suggestion
        raise ValueError(_(f'Invalid value for %(field)s field') % {"field": field})
```